sdk/include/aditof/status_definitions.h

```diff
@@ -11,11 +11,12 @@ namespace aditof {
  * @brief Status of any operation that the TOF sdk performs.
  */
 enum class Status {
-    OK,               //!< Success
-    BUSY,             //!< Device or resource is busy
-    UNREACHABLE,      //!< Device or resource is unreachable
-    INVALID_ARGUMENT, //!< Invalid arguments provided
-    GENERIC_ERROR     //!< An error occured but there are no details available.
+    OK,                    //!< Success
+    BUSY,                  //!< Device or resource is busy
+    UNREACHABLE,           //!< Device or resource is unreachable
+    INVALID_ARGUMENT,      //!< Invalid arguments provided
+    NOISE_REDUCTION_ERROR, //!< Noise reduction is not possible on far mode
```

Comment:
This is too specific for an error! We can't create an error for each feature we do.
Use the logger to print a more detailed message.
Use something like NOT_AVAILABLE or something like that, which can be reused.

Reply:
changed.

Thank you